tests/unit/models/test_model_interfaces.py

```diff
@@ -124,6 +124,15 @@ def _vgp(x: tf.Tensor, y: tf.Tensor) -> VGP:
     return m
 
 
+def _vgp2(x: tf.Tensor, y: tf.Tensor) -> VGP:
```

Comment:
@joelberkeley-secondmind this new model is necessary to test the update because the one with linear kernel has a very bad conditioning.